lib/auth/storage/storage.go

```diff
@@ -45,6 +45,10 @@ const (
 	statesPrefix = "states"
 	// idsPrefix is a key prefix for identities
 	idsPrefix = "ids"
+	// teleportPrefix is a key prefix to store internal data
+	teleportPrefix = "teleport"
+	// latestVersion is a key for storing latest version of teleport
```

Comment:
"latest" is a big ambiguous here. Is it the latest released version, the latest version running in the cluster, etc..

Should we call this `lastKnownVersion` or something similar?